.github/ISSUE_TEMPLATE/suggestion.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 'Suggestion'
```

Comment:
Feature request, rename everything. It matches with our tags

Reply:
done. However, I've kept [suggestion] as feature title (following contributing guide).